History.md

```diff
@@ -1,3 +1,9 @@
+unreleased
```

Comment:
@dougwilson is this how we update History.md for unreleased changes to the codebase?